examples/acl/config/dev/config.edn

```diff
@@ -30,18 +30,30 @@
 
  :framework.app/web-server        {:port  3000
                                    :join? false}
+ :acl/permissions                 {"posts"    [:read :create :update :delete :comment :react]
+                                   "comments" [:read :create :update :delete :reply :react]
+                                   "users"    [:read :create :update :delete :ban]}
```

Comment:
You collify the vals, add an example of single permission, like `"table" :single-permission`, to document this feature